### ISSUE TITLE
Keyboard issues

### ISSUE THREAD

**Issue by Airon90** [Tuxemon/Tuxemon#248]
1. On-screen keyboard should adapt by localization: if Italian localization is used (```it_IT```) Italian alphabet should be used (by now accented letters are missing), if Esperanto (```eo```) is used, Esperanto alphabet should be used (by now ĉapelitaj literoj are missing), if Arabic (```ar_XX```) is used, Arabic alphabet should be used (totally missing). Another solution is to implement ALL alphabets (missing Latin letters, all Greek, Arabic, Cyrillic, ... letters) and to divide all letters into "pages".

2. On Linux the combinations are not implemented: ```^``` + ```C``` results in ```Ĉ``` but it doesn't on Tuxemon

**Comment by bitcraft** [comment]
We'll need a way to list characters for each language.  Currently, it is stuck with English.  Can you test if you can enter the characters using the keyboard (not the game buttons)?

**Comment by Airon90** [comment]
> Can you test if you can enter the characters using the keyboard (not the game buttons)?

No, you can enter only letters or symbols that are listed on-screen.

> Pygame doesn't include advanced keyboard handling like ^ + C.

I thought that but I wanted a confirmation by a developer

> For reference, this is the font we use, and the glyphs that it supports. if a glyph is missing, then it will need to be added to the font and maybe make a custom font for Tuxemon.

Esperanto letters are present but IMHO they sucks. Thank you for the links!

**Comment by bitcraft** [comment]
I'll note here the the font is open source, so if anyone with a font editor wanted, new glyphs could be added.  Its unicode, so that is a nice start.
